decidim-meetings/config/locales/en.yml

```diff
@@ -107,6 +107,12 @@ en:
           edit:
             close: Close
             title: Close meeting
+        meeting_copies:
+          form:
+            select_open_type: Select an open type
```

Comment:
I feel like these sentences are not very clear, can you ask product for a better sentence? Open/closed are not types of "open", but different levels of accessibility to me 😕

Reply:
The related issue has some sentences explaining what "open", "public" and "transparent" concepts mean for a meeting. I'd add this as a help text to the input, at least.